tests/unit/file_lib_test.c

```diff
@@ -1614,6 +1614,24 @@ static void test_file_read(void)
     return_to_test_dir();
 }
 
+static void test_read_file_stream_to_buffer(void)
+{
+    setup_tempfiles();
+
+    {
```

Comment:
Why this sub-block?

Reply:
Copy paste from the other tests, just a way to be able to add multiple test cases with the same variable names reused.

Reply:
OK